man/autohbw.7

```diff
@@ -1,6 +1,6 @@
 .\" Automatically generated by Pandoc 2.9.2.1
 .\"
-.TH "AUTOHBW" "7" "2022-09-06" "AUTOHBW | MEMKIND Programmer's Manual"
+.TH "AUTOHBW" "7" "2022-11-02" "AUTOHBW | MEMKIND Programmer's Manual"
```

Comment:
Remove from this PR all man page files which don't have anything changed beside the date of creation. Only files which have their body edited/updated are relevant.